apps/web/scripts/kernel/handlers/common.ts

```diff
@@ -45,3 +53,31 @@ export const handleInstallLayout: HandleMethod = async ({
 		throw Error(error as string);
 	}
 };
+
+export const handleCheckInstalledLayout: HandleMethod = async ({
+	payload,
+	responseMethod,
+}) => {
+	const { requestId, id } = payload as InstallLayoutPayload;
+	const isInstalled = await checkInstalledExtensionById(id);
+
+	if (isInstalled) {
+		responseMethod(requestId, ResponseCode.SUCCESS);
+	} else {
+		responseMethod(requestId, ResponseCode.ERROR);
+	}
+};
```

Comment:
For convenience, we might want to make this method support dynamically like if without passing it will return all the installed layouts, users can check it. Or implement another method for it.

Reply:
In this case, I suppose each dapp only has 1 layout but yes there could have use case that check list layout user had installed